src/components/ScenarioParameters/components/ScenarioActions/ScenarioActions.spec.js

```diff
@@ -0,0 +1,135 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+/* eslint-disable react/prop-types */
+import React from 'react';
+import { ButtonTesting } from '../../../../../tests/MuiComponentsTesting';
+import { customRender } from '../../../../../tests/utils/renderInThemeAndStoreProviders';
+import { LaunchButton, SaveButton, DiscardChangesButton, StopRunButton } from './components/';
+import { getByDataCy } from '../../../../../tests/utils';
+import { SCENARIO_RUN_STATE } from '../../../../services/config/ApiConstants';
+import { useFormState } from 'react-hook-form';
```

Comment:
Do we need to import `useFormState` here? This module is mocked 4 lines below.

Reply:
In order to use `useFormState.mockReturnValue`inside several tests, it is required to import useFormState in addition with mock

Reply:
Ok, we can keep it then :smile: